bounties/level-2/sep10-javascript-server.md

```diff
@@ -0,0 +1,69 @@
+# JavaScript SEP-0010 Server Reference Implementation
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 1000 XLM | 1000 XLM | Capped (3)
+
+[//]: # (make sure to replace the file-name placeholders `BOUNTY_FILE_NAME_NO_EXTENSION` and `BOUNTY_FILE_NAME_WITH_EXTENSION` in the next two lines with the actual bounty filename)
+📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+label%3A%3Csep10-javascript-server%3E) for this bounty. \
+🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&link=https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-%3C2%3E/%3Csep10-javascript-server.md%3E) this bounty.
```

Comment:
The `<` and `>` around the placeholder need to as well. I'll fix the template (#102) as it is not obvious in the current state.
```suggestion
📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+label%3Asep10-javascript-server) for this bounty. \
🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&link=https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/sep10-javascript-server.md) this bounty.
```